src/test/kotlin/day03/HeroTest.kt

```
@@ -1,29 +1,33 @@
+    @org.junit.After
+    fun tearDown() {
+    }
+    // 테스트 수행 후에 수행되는 부분
 
-@org.junit.Test
- fun sleep() {
-   // 준비
-   val hero = Hero("홍길동", 10)
+    @org.junit.Test
+    fun attack() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)